### DISCUSSION TITLE
Tremendous work!

### DISCUSSION THREAD

**Discussion by Mikachu2333** [crusty-pie/toolchain#199] [General]
Thanks a thousand times for your work, which helped me get rid of the fucking GitHub action configuration job as a noob!!!

**Comment by placintaalexandru** [comment]
Thank you for appreciation @Mikachu2333. I guess the original maintainer who archived the action deserves part of the credit as well. I forked it and tried to address some issues.

**Reply by Mikachu2333** [reply]
Yes, you're right, the original author did a lot of excellent work as well! 
I've uesed https://github.com/actions-rs/toolchain , but GitHub warned me that some packages in it must be updated and I'm at a loss what to do with the cumbersome configs, and, I fonud your fork.\
RE-thanks for your efforts
